app/models/queue_item.rb

```diff
@@ -5,22 +5,45 @@ class QueueItem < ActiveRecord::Base
   delegate :category, to: :video
   delegate :title, to: :video, prefix: :video
 
-  before_create :set_order
+  before_create :set_position
 
-  default_scope { order(:order) }
+  validates_numericality_of :position, only_integer: true, allow_blank: true
+  validates_uniqueness_of :video, scope: :user
+
+  def review
+    @review ||= Review.where(author: user, video: video).first
```

Comment:
Ooooh that makes sense. 
I guess this is because I'm working on a Rails 3.0.9 project at my work. I've checked countless times if I could it differently in Rails 3 :-).

Thanks for the tip!